hydra/core/utils.py

```diff
@@ -186,6 +187,9 @@ def run_job(
                 ret.return_value = task_function(task_cfg)
                 ret.status = JobStatus.COMPLETED
             except Exception as e:
+                name = "HYDRA_FULL_ERROR"
+                if name in os.environ and os.environ[name] == "1":
+                    log.info(traceback.format_exc())
```

Comment:
better `log.error` here, no?

Reply:
yeah, I agree. also, it is likely crash from here, so maybe that won't be missed in most cases.

Reply:
I've created followup PR https://github.com/facebookresearch/hydra/pull/2549 changing the log level from `INFO` to `ERROR`.